rustls/src/server/server_conn.rs

```diff
@@ -154,6 +154,20 @@ impl<'a> ClientHello<'a> {
     /// Get the alpn.
     ///
     /// Returns `None` if the client did not include an ALPN extension
+    ///
+    /// ALPN (Application Layer Protocol Negotiation) contains a set of identifiers
+    /// which are encoded as a set of octet values, which could be UTF-8 encoded.
+    /// It's a form of TLS extension allowing the client to provide an externally visible
+    /// marker for the application-layer protocol associated with the TLS connection.
+    ///
+    /// Common examples of ALPN identifiers given by web clients are "http/1.1", "h2" and "h3",
```

Comment:
I would probably not mention h3 here, since it has a more complicated situation (since H3 is built on QUIC, it doesn't use TCP).